rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/run/configurations/unityExe/UnityExeConfigurationType.kt

```diff
@@ -0,0 +1,17 @@
+package com.jetbrains.rider.plugins.unity.run.configurations.unityExe
+
+import com.intellij.execution.configurations.ConfigurationTypeBase
+import com.intellij.util.IconUtil
+import icons.UnityIcons
+
+class UnityExeConfigurationType : ConfigurationTypeBase("RunUnityExe", "Unity Executable",
+    "Unity executable configuration", IconUtil.scale(UnityIcons.RunConfigurations.UnityExe, null, 0.33f)) {
```

Comment:
Perhaps these also need changing to "Unity Standalone Player". `RunUnityExe` looks like a persisted ID, worth checking and commenting that we shouldn't change it.